finality-aleph/src/testing/data_io.rs

```diff
@@ -60,6 +60,10 @@ impl<B: BlockT> RequestBlocks<B> for TestBlockRequester<B> {
             .unbounded_send(AlephData { hash, number })
             .unwrap();
     }
+
+    fn clear_justification_requests(&self) {
+        panic!("`clear_justification_requests` not implemented!")
```

Comment:
there's already `unimplemented!()` macro?

Reply:
I know, but I wanted to be consistent with rest of the repo (we never use this macro and write this in this way)